README.md

```diff
@@ -33,6 +33,39 @@ and the API is subject to change._
    ion help
    ```
 
+### Docker Instructions
+
+1. Clone the repository (recursive clone not necessary)
+   ```
+   git clone https://github.com/amzn/ion-cli.git
+   ```
+2. Step into the newly created directory
+   ```
+   cd ion-cli
+   ```
+3. Install Docker (see OS specific instructions on the [Docker website](https://docs.docker.com/get-docker/))
```

Comment:
I'd make this step 1. If the host system needs other libraries (Debian needs the host to have `cmake` and `clang`, you mentioned?) then you could turn this step into `Install Dependencies` with a sublist.

Reply:
Good point, this should be re-ordered. 

For the Docker specific instructions, the only host dependency will be Docker. Sorry it was confusing how I mentioned the `clang` and `cmake` deps. I wanted to call out that the host system appears to only need those.